docs/getting-started/installation.md

```diff
@@ -251,3 +251,18 @@ podAnnotations: {}
 [ecr]: https://gallery.ecr.aws/aquasecurity/trivy
 [registry]: https://github.com/orgs/aquasecurity/packages/container/package/trivy
 [helm]: https://github.com/aquasecurity/trivy/tree/{{ git.tag }}/helm/trivy
+
+
+## pre-commit hook
+
+Add this to your [pre-commit](https://pre-commit.com/) `.pre-commit-config.yaml` config:
+
+```yaml
+repos:
+-   repo: https://github.com/aquasecurity/trivy.git
+    rev: {{ git.tag }}
+    hooks:
+    -   id: trivyfs
+```
+
+According to [pre-commit](https://pre-commit.com/#golang) you will require [go](https://go.dev/) to be installed
```

Comment:
`pre-commit ` is developer helper, like `golangci-lint`.
I am not sure that we need to add instruction to docs.

Reply:
I thought as developer if I wanted to know on how to get started integrating trivy as a pre-commit hook I would look in the "getting started / installation" section

Reply:
Now i understand. You're right. This is a good place for it.